client_configuration.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+
+package gosnowflake
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// log levels for easy logging
+const (
+	Off   string = "OFF"   // log level for logging switched off
```

Comment:
I changed to lowercase, but I had to add prefix level, because we cannot have const "error" because it is a reserved word. So I changed it to levelOff, levelError, etc.